lib/Backends/Interpreter/InterpreterDeviceManager.cpp

```diff
@@ -84,6 +84,8 @@ void InterpreterDeviceManager::addNetworkImpl(const Module *module,
   for (const auto &func : functions) {
     if (func.second->getRuntimeBundle().getConstants() == nullptr) {
       func.second->getRuntimeBundle().collectConstants(module);
```

Comment:
I think you can remove this call, since InterpreterFunction::collectConstants calls the method on the bundle.